sem_final/models/product.py

```diff
@@ -0,0 +1,14 @@
+from pydantic import BaseModel, Field
+
+
+class Product(BaseModel):
+    id: int
+    title: str = Field(..., title="Title", max_length=100)
+    description: str = Field(default='', title="Description", max_length=300)
+    price: float = Field(..., title="Price", gt=0, le=1_000_000)
+
+
+class ProductIn(BaseModel):
+    title: str = Field(..., title="Title", max_length=100)
+    description: str = Field(default='', title="Description", max_length=300)
+    price: float = Field(..., title="Price", gt=0, le=1_000_000)
```

Comment:
Классы почти одинаковые, значит один можно наследовать от другого
```suggestion
class ProductIn(BaseModel):
    title: str = Field(..., title="Title", max_length=100)
    description: str = Field(default='', title="Description", max_length=300)
    price: float = Field(..., title="Price", gt=0, le=1_000_000)
    
class Product(ProductIn):
    id: int
```